arborist/server.go

```diff
@@ -651,19 +651,7 @@ func (server *Server) handlePolicyCreate(w http.ResponseWriter, r *http.Request,
 	_ = jsonResponseFrom(created, 201).write(w, r)
 }
 
-func (server *Server) handlePolicyOverwrite(w http.ResponseWriter, r *http.Request, body []byte) {
-	policy := &Policy{}
-	err := json.Unmarshal(body, policy)
-	if err != nil {
-		msg := fmt.Sprintf("could not parse policy from JSON: %s", err.Error())
-		server.logger.Info("tried to create policy but input was invalid: %s", msg)
-		response := newErrorResponse(msg, 400, nil)
-		_ = response.write(w, r)
-		return
-	}
-	// Overwrite policy name from json with policy name from query arg.
-	// After 3.0.0, when PUT /policy is deprecated and only PUT /policy/{policyID} is allowed,
-	// can remove the !="" check. For now, if policy name not found in url, default to name in json.
+func (server *Server) overwritePolicy(w http.ResponseWriter, r *http.Request, policy Policy) {
 	if mux.Vars(r)["policyID"] != "" {
```

Comment:
you should put this comment back, _or_ remove the `if` statement and [this line](https://github.com/uc-cdis/arborist/blob/3.0.0/arborist/server.go#L108-L109)